js/style/style_batch.js

```diff
@@ -217,4 +230,15 @@ styleBatch.prototype = {
     }
 };
 
+function filtersEqual(a, b) {
```

Comment:
This looks like an `isDeepEqual` method that works only on arrays. Can we make this a general purpose `util` method instead?